main.tf

```diff
@@ -0,0 +1,117 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 3.8"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+
+  subscription_id = "d33b95c7-af3c-4247-9661-aa96d47fccc0"
+  client_id       = "0568731c-73ad-44fa-89b6-8123ee27d574"
+  client_secret   = var.arm_client_secret
+  tenant_id       = "7d6f97d6-d755-4c10-b763-409cc4b6638f"
```

Comment:
Typically we don't want to hardcode these but provide them as environment variables (or just be logged into the AZ client anyway) since otherwise it takes a code change to look at different environments etc.

Reply:
Removed and moved to environments vars.